docs/script-catalog/scim/scim.md

```
@@ -120,74 +120,74 @@ These methods are called if `getApiVersion` returns a number >= 5.
 With the `manageResourceOperation` and `manageSearchOperation` methods you can make complex decisions on how processing should take place based on contextual data and the incoming payload.
 
 **manageResourceOperation**
-  
+
 This method is invoked when any following operations are executed: resource creation, modification, removal and retrieval by ID. In case of bulks, the method is called for every operation that fits into these categories.
 
 Parameters are described in the table below:
-  
+
 |Name|Description|Class/Link|
 |:---|:---|:---
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)